src/Umbraco.Web.UI.Client/src/views/components/users/change-password.html

```diff
@@ -6,19 +6,15 @@
     </div>
     <div ng-switch="vm.changing">
         <div ng-switch-when="false">
-            <a href="" ng-click="vm.doChange()" class="btn btn-small">
+            <a href="" ng-click="vm.doChange()" class="btn umb-button__button btn-action umb-button--">
```

Comment:
Hey @poornimanayar - It would be awesome if you could also change this to `<buttton type="button" ng-click="vm.doChange()" class="btn umb-button__button btn-action">...</button>`

Maybe you could even have a look at using the `<umb-button>` component/directive instead. There are a few examples on how this can be setup if you search the source code. Only thing I can't tell currently is what the styling needs to look like in this scenario... :-)